TypeSystem/Schema/golden/smoke_test_struct.cc

```diff
@@ -69,7 +69,7 @@ struct TemplatedInheriting_T9200000002835747520 : A {
 };
 struct Templated_T9209626390174323094 {
   int32_t foo;
-  TemplatedInheriting_T9200000002835747520 bar;
+  TemplatedInheriting_Z bar;
```

Comment:
@dkorolev There's no declaration of `TemplatedInheriting_Z` in this file. How it supposed to work? :-/